configs/unrolled_recon.yaml

```diff
@@ -90,12 +92,12 @@ simulation:
 #Training
 
 training:
-  batch_size: 16
-  epoch: 10
+  batch_size: 8
+  epoch: 50
```

Comment:
would it make sense to add a parameters for training set size? Where `null` uses whole dataset

Reply:
It already exist in the dataset part of the config.